user_tools/src/spark_rapids_pytools/common/utilities.py

```diff
@@ -53,7 +53,7 @@ def gen_uuid_with_ts(cls, pref: str = None, suffix_len: int = 0) -> str:
         :param suffix_len:
         :return:
         """
-        ts = datetime.datetime.utcnow().strftime('%Y%m%d%H%M%S')
+        ts = datetime.datetime.now().astimezone().strftime('%Y%m%d%H%M%S')
```

Comment:
The `datetime.datetime.utcnow()` is deprecated.
The new code uses the timeZone of the system instead of the UTC. This is more aware compared to previous naive implementation.